app/js/components/queryBuilder/queryParamSelector/queryParamSelector.tpl.html

```diff
@@ -1,42 +1,31 @@
 <header class="row dashboard-column-header">
 	<h2>Parameter Selector</h2>
 </header>
-
 <div class="row query-param-selector-ctrls">
-
-	
-
 	<div class="col-lg-1 results-count" >
-
 		<small><b>results</b></small>
-
 		<span 
 			ng-if="$ctrl.isSearching"
 			class=" glyphicon glyphicon-refresh" 
-  		></span> 
-
+			></span> 
```

Comment:
It probably makes more sense if the indentation of this line matches that on line 7.